specs/disciplina/witness/api/explorer.yaml

```diff
@@ -0,0 +1,215 @@
+swagger: "2.0"
+info:
+  description: "This is an HTTP API for interacting with Disciplina Witness node. It is used by Block Explorer frontend to request info about blocks, addresses and transactions."
+  version: 0.1.0
+  title: Disciplina Block Explorer API
+  termsOfService: "https://disciplina.io/tnc.pdf"
+  contact:
+    email: hi@serokell.io
+  license:
+    name: Apache 2.0
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "localhost:8091"
+basePath: /api/witness/v1
+schemes:
+  - https
+  - http
+paths:
+  /blocks:
+    get:
+      summary: >-
+        Get list of blocks.
+        Only the batch of N most recent blocks are returned at once.
+        The hash of the last block in the batch can be passed as `before` parameter to get the next batch.
```

Comment:
It may be my and some other guys' subjective opinion, but `summary` is usually one-sentence note, remaining goes to `description`.

Reply:
oh, right `description`